### PR TITLE
pangraph-0.2.0

### PR DESCRIPTION
**Pull request by thisiswhereitype**
This was to be patch 0.1.3 but I explored making the `Protograph` and changes to `Edge` which resulted in this needing a major bump. So workcraft will be 0.2.1.

pangraph-0.2.0 
* Addition of conversion and revert for FGL.
* Addition of Parser, Serializer and AST for GML.
* Change `Edge` types to now only construct with a `VertexID` type.
* Change `edgeEndpoints` to return VertexID.

Further internally there is now the following modules:
```haskell
-- A module to ease compatibility between Hexml and Pangraph.
Pangraph.Internal.HexmlExtra
-- A module containing boilerplate code for constructing of pangraphs.
Pangraph.Internal.ProtoGraph
```

### CODE REVIEW

File: src/Pangraph/ProtoGraph.hs [hunk 1]
```diff
@@ -0,0 +1,87 @@
+{-# LANGUAGE OverloadedStrings #-}
```
Comment (snowleopard) [inline src/Pangraph/ProtoGraph.hs]: It would be nice to add some top-level module documentation to describe the purpose of various modules.

**Comment by snowleopard** [comment]
@thisiswhereitype What is the motivation behind `Protograph`? Isn't `Pangraph` sufficient? Having two seemingly similar datatypes in the library may be confusing. If `Protograph` is merely a convenient intermediate representation, it should perhaps live in an `Internal` module, with a warning that it is not a part of the API.

**Comment by thisiswhereitype** [comment]
I have moved it as you suggest to an internal module and refactored. I have also improved the quality of the docs on all exposed modules. @snowleopard

**Review by snowleopard — APPROVED** [review]
Looks better now! Feel free to merge when ready.

**Comment by thisiswhereitype** [comment]
Merged thanks for the review!

**Pull request merged**